src/patternfly/components/Tile/tile.scss

```diff
@@ -0,0 +1,149 @@
+.pf-c-tile {
+  --pf-c-tile--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-tile--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-tile--PaddingBottom: var(--pf-global--spacer--lg);
+  --pf-c-tile--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-tile--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-tile--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-tile--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tile--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-tile--hover--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-tile--m-selected--BorderWidth: var(--pf-global--BorderWidth--md);
+  --pf-c-tile--m-selected--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-tile--disabled--BackgroundColor: var(--pf-global--disabled-color--300);
+
+  // title
+  --pf-c-tile__title--Color: var(--pf-global--Color--100);
+  --pf-c-tile--hover__title--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--m-selected__title--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--disabled__title--Color: var(--pf-global--disabled-color--100);
+
+  // icon
+  --pf-c-tile__icon--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-tile__icon--FontSize: var(--pf-global--icon--FontSize--md);
+  --pf-c-tile__icon--Color: var(--pf-global--Color--200);
+  --pf-c-tile--m-selected__icon--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--hover__icon--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile__title--m-stacked__icon--MarginBottom: var(--pf-global--spacer--xs);
+  --pf-c-tile__title--m-stacked__icon--FontSize: var(--pf-global--icon--FontSize--lg);
+  --pf-c-tile__title--m-stacked__icon--m-large--FontSize: var(--pf-global--icon--FontSize--xl);
+  --pf-c-tile__title--m-stacked__icon--m-large--Width: var(--pf-c-tile__title--m-stacked__icon--m-large--FontSize);
+  --pf-c-tile__title--m-stacked__icon--m-large--Height: var(--pf-c-tile__title--m-stacked__icon--m-large--Width);
+  --pf-c-tile__title--m-stacked__icon--Color: var(--pf-global--Color--100);
+  --pf-c-tile--hover__title--m-stacked__icon--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--m-selected__title--m-stacked__icon--Color: var(--pf-global--primary-color--100);
+  --pf-c-tile--disabled__title--m-stacked__icon--Color: var(--pf-global--Color--200);
+
+  // subtext
+  --pf-c-tile__subtext--Color: var(--pf-global--Color--100);
+  --pf-c-tile__subtext--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-tile--disabled__subtext--Color: var(--pf-global--disabled-color--100);
+
+  position: relative;
+  display: inline-block;
```

Comment:
I think I'd use `display: inline-grid` here, then apply a `row-gap`. That way, in the future, if we want to use `subgrid` to manage row height in the title or text when tiles are adjacent, we could do that.